Server/src/main/java/net/simon987/server/assembly/DefaultInstructionSet.java

```diff
@@ -104,18 +104,9 @@ public Instruction get(int opcode) {
             return defaultInstruction;
         }
     }
-
     /**
-     * Add a new instruction to the instructionSet
-     *
-     * @param opcode      opcode of the instruction
-     * @param instruction Instruction to add
+     * Get an instruction frim its mnemonic
```

Comment:
from

Reply:
Done. Sorry about that. Did it quick and dirty in Vim and didn't have spellcheck turned on.